src/app/static/src/app/components/Stepper.vue

```diff
@@ -37,7 +37,7 @@
         </div>
         <div v-if="!loading" class="content">
             <div class="pt-4">
-                <adr-key v-if="isActive(1) || isActive(2)"></adr-key>
+                <adr-integration v-if="isActive(1)"></adr-integration>
```

Comment:
I think it only makes sense to show this on step 1, since it's supposed to be sort of step 0 (and editing it on step 2 would potentially invalidate the step you were on.)